app/views/contexts/hub_context.py

```diff
@@ -85,13 +101,17 @@ def get_row_context_for_section(
                                 section_name=section_name
                             ),
                             "url": section_url,
+                            "attributes": {"data-qa": "summary-actions-summary-button"},
```

Comment:
I realise this was the value of the attribute previously, but this would be better named something like `summary-actions-section-link`.